### PR TITLE
Added tests for Dropdown

### LINKED ISSUE #937: Add tests for Dropdown

### PR DESCRIPTION
**Pull request by prasidhprabudhan**
Fixes #937

### CODE REVIEW

**Comment by prasidhprabudhan** [comment]
@ajmaln _a
Please review.

**Comment by prasidhprabudhan** [comment]
@prasidhprabudhan _a
Some improvements required.

**Comment by prasidhprabudhan** [comment]
@ajmaln _a
Please review.

File: tests/Dropdown.test.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  });
+
+  it("should render all the options when clicking dropdown without error", async () => {
+    const { getByText } = render(
+      <Dropdown label="Dropdown" >
+        {options}
+      </Dropdown>
+    )
+    userEvent.click(getByText("Dropdown"))
+    const listItems = await screen.findAllByRole("listitem")
```
Comment (ajmaln) [inline tests/Dropdown.test.js:31]: Lets query by option text here as it will also help us verify the option text is also rendered.

**Review by ajmaln — COMMENTED** [review]
@prasidhprabudhan _a A minor suggestion

**Comment by prasidhprabudhan** [comment]
@ajmaln _a
Please review.

**Review by ajmaln — APPROVED** [review]

**Pull request merged**